server/api/movies.ts

```diff
@@ -85,6 +87,68 @@ MoviesRouter.post('/', (req: Request, res: Response) => {
     });
 });
 
+MoviesRouter.get('/moviesByRatingPG', (req: Request, res: Response) => {
+  grabMovieIdWithRating("PG")
+     .then((data: any) => {
+       res.send(data);
+     }).catch((error: string) => {
+       console.log(error);
+       res.status(500).end();
+     });
+   });
+
+   MoviesRouter.get('/moviesByRatingPG-13', (req: Request, res: Response) => {
+  grabMovieIdWithRating("PG-13")
+     .then((data: any) => {
+       res.send(data);
+     }).catch((error: any) => {
+       console.log(error);
+       res.status(500).end();
+     });
+   });
+
+   MoviesRouter.get('/moviesByRatingR', (req: Request, res: Response) => {
+  grabMovieIdWithRating("R")
+     .then((data: any) => {
+       res.send(data);
+     }).catch((error: any) => {
+       console.log(error);
+       res.status(500).end();
+     });
+   });
+
+   MoviesRouter.get('/moviesByRatingNC-17', (req: Request, res: Response) => {
+  grabMovieIdWithRating("NC-17")
+     .then((data: any) => {
+       res.send(data);
+     }).catch((error: any) => {
+       console.log(error);
+       res.status(500).end();
+     });
+   });
```

Comment:
All of these `movieByRating` GET handlers should be combined into one function.  Use a parameter.